scripts/python_installer

```diff
@@ -105,11 +107,17 @@ function install_python() {
     echo_success_message " - Python $PYTHON_VERSION is installed at $PYENV_BIN"
 
     if python_is_not_in_path; then
-        print_path_export_instructions
+        if [[ "$SURPRESS_PATH_EXPORT_MESSAGE" -ne "true" ]]; then
```

Comment:
`-e "false"` would have been preferable, but if you use booleans to represent truthiness, you can simplify as:

```sh
if [ $SURPRESS_PATH_EXPORT_MESSAGE ] ; then
...
```

Reply:
This makes sense, to add some clarity to what it's doing though I may change the variable name from $SURPRESS_PATH_EXPORT_MESSAGE to $SHOW_PATH_EXPORT_MESSAGE because if we want to test for truthiness then the variable should reflect that.

Reply:
Gave this a shot and it looks like this works only without the brackets. So the statement will be if ${SHOW_PATH_EXPORT_MESSAGE} rather than if [ ${SHOW_PATH_EXPORT_MESSAGE} ]